src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -445,7 +445,7 @@ public ValueTask<(bool success, MessageEnvelope envelope)> TryPeekOneAsync(TimeS
         {
             return ReceiveWhileAsync(max, filter, msgs, cancellationToken)
                 .ToListAsync(cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.